ScnSoft-Education/LeetCodeTraining.Tests/4SumTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿using LeetCodeTraining.FourSum;
+
+namespace LeetCodeTraining.Tests
+{
+    public class FourSumSolutionTests
+    {
+        public static IEnumerable<object[]> FourSumTestCases()
+        {
+            yield return new object[]
+            {
+                new int[] { 1, 0, -1, 0, -2, 2 }, 0,
+                new int[][] { new int[] { -2, -1, 1, 2 }, new int[] { -2, 0, 0, 2 }, new int[] { -1, 0, 0, 1 } }
```

Comment:
Please do 2nd array's elements ad List<IList<int>>, in this case you will be able to do the test assortment in one line as:
Assert.Equal(expected, result);